Makefile

```diff
@@ -5,11 +5,11 @@
 
 sagefile := .sage/bin/sagefile
 
-.PHONY: $(sagefile)
-$(sagefile):
+$(sagefile): .sage/go.mod $(shell find .sage/.. -type f -name '*.go')
```

Comment:
Why the `..` after `.sage` - is that intentional?

Reply:
I added what was formerly there. (originally added in https://github.com/einride/sage/commit/296ceab0d84a66543506b5374d11dc1bc92bc47c#diff-23e713a554f536f060bdf56c6960e598d68657017b92b1466b49339ec040f061R2 ) . My feel is that as this is the sage repository, there should be more go files one level up. 

I honestly find it hard to argue why `$(shell find .sage/.. -type f -name '*.go')` will add any value. If find does not resolve any files, it will be expanded to nothing, which is fine for make. If find does resolve anything those files will by definition be there. 

What is it I am missing? or shall we remove that clause?

Reply:
@odsod What do you think?

Reply:
Hmm, I was thinking that it's only the Go files in the `.sage` module that impact if the need to rebuild the sagefile. The actual source code changing should not trigger a rebuild of the build tooling - or am I missing something? So it should `find` from the root of the sage module, but not one level above?

Reply:
We look for all go files in the repository for this repository only, it's a conditional in the generate code. This is because this repo is special :) and as you can see in the go mod in the sage folder, we have a local replace.

We could simplify it by just always rebuilding in this repo. So rewrite the conditional in the generate file to add a Phony target instead.

Reply:
:+1: makes sense, cleared the dots.

Reply:
Great!